openapi/components/schemas/Invoices/InvoiceItem.yaml

```
@@ -4,54 +4,60 @@ required:
   - unitPrice
 properties:
   id:
-    description: The website identifier string.
+    description: |-
+      Unique website identifier string. A website is where an
+      organization obtains a customer through a subscription. For more
+      information, see
+      [Obtain an organization ID and website ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
     readOnly: true
     allOf:
       - $ref: ../ResourceId.yaml
   type:
-    description: Invoice item's type.
+    description: Type of invoice item.
     type: string
     x-basic: true
     enum:
       - debit
       - credit
   description:
-    description: Invoice item's description.
+    description: Description of the invoice item.
     type: string
   unitPrice:
-    description: Invoice item's price.
+    description: Price of the invoice item.
```

```suggestion
    description: Unit price of the invoice item.
```
Unit price is the price for one unit as opposed to taking quantity into account.